### ISSUE TITLE
Draft operational guidance and VFS-facing guidance on Core CC template

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#36086]
## User Story
As a Governance team member, I want detailed guidance on the Core Collaboration Cycle ticket process so that I understand the changes and my responsibilities.

As a VFS team, I want clear guidance provided on the Core Collaboration Cycle ticket process so that I understand the changes, my responsibilities and how to engage with the Collaboration Cycle.

- Intended Audience: Governance Team
- Desired Outcome: Shared team understanding of internal changes
- Due Date: Drafts prepared prior to 4/26 research readout

## Description
With the switch to the new Core Collaboration Cycle ticket template in GitHub, VFS teams are going to need guidance on how the Core CC template works in comparison to the old process. In addition to updating the Platform Website touchpoint pages that impact the request process, VFS teams will need guidance detailing the new process. Ideally, this can be the Collaboration Cycle Kickoff page as this is the first interaction point in the Collaboration Cycle.

In parallel, Governance team operational guidance should also be updated to reflect the new process. 

The updated Collaboration Cycle kickoff page should continue to have the necessary kickoff questions, but also explain the purpose, value and benefits of the Core CC template, how requesting reviews will work, how labels will work, the Outcome of each review (AKA feedback...this can point to the specific touchpoint pages), closing/tracking the ticket, and any other information VFS teams will need related to the Core CC template and the Collaboration Cycle.

**Note: Do not publish any of the modified pages when this ticket is complete. .**

**Note # 2: Throughout this initiative, we have referred to the new ticket as the "Core Collaboration Cycle ticket". However, once released, we will drop the word "core" and the ticket will be referred to as the "Collaboration Cycle ticket" or "Collaboration Cycle github ticket." Please refrain from using the word "Core" in VFS-facing guidance and operational guidance as it is unnecessary.**

Additional background for [Collaboration Cycle - Improve tracking and status visibility of VA.gov products](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/34697) initiative <br />

## Artifacts Needing Updates
(list of artifacts needing updated to be determined by Governance team members post creation of swim lane user flow)

## Tasks
- [x] Create swim lane/user flow of [entire Collab Cycle process](https://app.mural.co/t/adhocvetsgov9623/m/adhocvetsgov9623/1650382660395/9efc51c8f8983a2b73c7903d9afe5f617aa799ab?sender=u045df875f30f7e9f34339908) from start to finish. 
- [x] When swim lane/user flow is created/finalized, ask the following question(s) about each action within the flow:
   - Where does *User* find guidance/documentation to support the necessary task?
   - Identify if the guidance/documentation is operational (aka only for Governance team), VFS-facing (only for VFS) or both.

- [ ] Review and update if needed:
  - [x] Governance team - [Collaboration Cycle Kickoff - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1749286928/Collaboration+Cycle+Kickoff+-+Moderator+Responsibilities) -Allison
   - [x] Governance team - [Design Intent - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1748336706/Design+Intent+-+Moderator+Responsibilities)  -Allison
   - [x] Governance team - [How to provide feedback for a Design Intent](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1934426130/How+to+provide+feedback+for+a+Design+Intent)  -Jim
   - [x] Governance team - [Midpoint Review - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1821442052/Midpoint+Review+-+Moderator+Responsibilities)  -Allison
   - [x] Governance team - [How to provide feedback for a Midpoint Review](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1945239838/How+to+provide+feedback+for+a+Midpoint+Review)   -Jim
   - [x] Governance team - [Staging Review - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1819476255/Staging+Review+-+Moderator+Responsibilities)  -Allison
   - [x] Governance team - [How to document feedback for Staging Review](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1839530098/How+to+document+feedback+for+Staging+Review)   -Jim

- [x] Review and update these VA.gov platform pages if needed:
    - [x] [Collaboration Cycle Kickoff](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Collaboration-Cycle-Kickoff.1760493650.html)  -Allison
    - [x] [Touchpoints](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Touchpoints.1768423528.html) -Allison
    - [x] [Design Intent](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Design-intent.1782153235.html)  -Cheryl
    - [x] [Sitewide Content and IA intake request](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Sitewide-Content-and-IA-intake-request.2124906551.html)  -Cheryl
    - [x] [Research review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Research-review.1781891143.html)  -Cheryl
    - [x] [Midpoint review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html)  -Cheryl
    - [x] [Staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html)  -Cheryl
      - [x] [Anatomy of a Staging Review Issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html)  -Cheryl
- [x] Review Touchpoint Ticket Content
  - [x] [Design Intent ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=governance-team%2C+collaboration-cycle%2C+collab-cycle-review%2C+design-intent&template=design-intent-collaboration.md&title=Design+intent+%5BTeam+Name+-+Feature+Name%5D) -Jim
  - [x] [Midpoint Review ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=governance-team%2C+midpoint-review%2C+collaboration-cycle%2C+collab-cycle-review&template=midpoint-review.md&title=Midpoint+Review+%5BTeam+Name%2C+Feature+Name%5D)  -Jim
  - [x] [Staging Review ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=governance-team%2C+collaboration-cycle%2C+staging-review%2C+collab-cycle-review&template=staging-review.md&title=Staging+Review+%5BTeam+Name+-+Feature+Name%5D) -Jim

- [x] Add phase-out disclaimer to individual GH tickets 

## Acceptance Criteria
- [ ] Governance team members are familiar with the new process for the core CC and understand their roles and responsibilities
- [ ] VFS team members have received updated guidance on how to engage with the Collaboration Cycle through the new core CC process
- [ ] Updates have been Peer Reviewed by Governance Team member
- [ ] User Story need has been fulfilled

---

## Related Governance Team Resources
_For additional information and guidance to complete tasks, please see links below._
[Documentation Update Guidance for GitHub templates](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2047868956/Guidance+for+updating+GitHub+Issue+templates) <br />
[Documentation Update Guidance for Platform Website pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089091115/Guidance+for+updating+Platform+Website+pages) <br />
[Documentation Update Guidance for Confluence pages](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2089222173/Guidance+for+updating+Confluence+pages) <br />
[Guidelines for Governance Team Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1127219219/Guidelines+for+Team+spaces) <br />
[Guidelines for Platform Products and Services Confluence Space](https://vfs.atlassian.net/wiki/spaces/AP/pages/1321140361/Guidelines+for+Platform+Products+and+Services+space) <br />
[Description of Platform Website spaces and content](https://vfs.atlassian.net/wiki/spaces/AP/pages/1241579796/Organization) <br />

---
## How to configure this issue
- [ ] **Ensure that story is attached to appropriate Iniative (if not tech debt)** (what body of work is this a part of?)
- [ ] Ensure that any additional labels are added ('tech-debt', 'collaboration-cycle')
- [ ] Ensure that story is in the correct pipeline (Backlog, Planned, In Progress, Review/QA, Closed)
- [ ] Ensure that assignees are correct so you receive all updates/comments on the ticket

**Comment by shiragoodman** [comment]
@chaseakirby can we update this ticket please?

**Comment by allison0034** [comment]
@CherylEvans @jimuhh I have divided up the pages evenly. If your content is going to affect another page just make sure to sync with that person so we are using the same language. Our goal is to have these done EOD Tuesday. Dont forget to follow the new guidelines for scrollviewport on platform website and do not publish them, keep them in draft mode until we get approval. : )

**Comment by jimuhh** [comment]
Thanks @allison0034. I'm clear on what to look for on the assigned Governance Team pages but less clear on what needs to happen w/ the touchpoint GH tickets since those are being phased out in favor of the consolidated ticket.

**Comment by shiragoodman** [comment]
@jimuhh @allison0034 @CherylEvans I'd like to request that you create a documentation draft for every page specified above (VFS-facing and Operational) and put them here: https://vfs.atlassian.net/wiki/spaces/PPST/pages/2165440978/Core+Ticket+guidance+36086 This is so when we walkthrough the full process, it will be easy to identify which documentation drafts we need to reference.

Let me know if you have any questions -thanks!

**Comment by CherylEvans** [comment]
Updated Design Intent page Request Section: [Link](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2167767138/Cheryl+-+Copy+of+Design+intent#Request)

**Comment by jimuhh** [comment]
Did the same on Midpoint review document:

> If you don’t have any feedback for this touchpoint, comment in the core github ticket epic with “I have no Midpoint Review feedback” and include your practice area.

**Comment by jimuhh** [comment]
on [Jim- How to document feedback for Staging Review](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2169241650/Jim-+How+to+document+feedback+for+Staging+Review)
No changes.

**Comment by jimuhh** [comment]
I reviewed the CC tickets. Can we make adding the "ticket being phased out" note a separate ticket, since it's going to involve template edits, pull requests and a certain amount of collaborative word-smithing to complete?

**Comment by jimuhh** [comment]
I made and assigned the pull requests for Design Intent and Staging.

**Comment by allison0034** [comment]
@jimuhh what are you doing a pull request for? I believe (and @chaseakirby correct me if I am wrong) we are going to take Clayton's notes from the walk thru and assign new tickets to people.

**Comment by chaseakirby** [comment]
Moved phase-out disclaimer to touchpoint respective tickets as discussed. Closing ticket.
